src/rdkafka_request.c

```diff
@@ -5484,14 +5489,21 @@ rd_kafka_resp_err_t rd_kafka_DescribeConfigsRequest(
                         /* config_name */
                         rd_kafka_buf_write_str(rkbuf, entry->kv->name, -1);
                 }
+                rd_kafka_buf_write_tags_empty(rkbuf);
         }
 
 
-        if (ApiVersion == 1) {
+        if (ApiVersion >= 1) {
                 /* include_synonyms */
                 rd_kafka_buf_write_i8(rkbuf, 1);
         }
 
+        if (ApiVersion >= 3) {
+                /* include_documentation */
+                rd_kafka_buf_write_i8(rkbuf, 1);
```

Comment:
Why always true?

Reply:
removed it since wasn't neccesarry as we are not upgrading the api version